doc/Motr_LNet_Transport.rst

```diff
@@ -0,0 +1,625 @@
+==============================
+HLD OF Motr LNet Transport
+==============================
+
+This document presents a high level design (HLD) of the Motr LNet Transport. The main purposes of this document are: (i) to be inspected by M0 architects and peer designers to ascertain that high level design is aligned with M0 architecture and other designs, and contains no defects, (ii) to be a source of material for Active Reviews of Intermediate Design (ARID) and detailed level design (DLD) of the same component, (iii) to serve as a design reference document.
+
+***************
+Introduction
+***************
+
+The scope of this HLD includes the net.lnet-user and net.lnet-kernel tasks described in [1]. Portions of the design are influenced by [4].
+
+***************
+Definitions
+***************
+
+- Network Buffer: This term is used to refer to a struct M0_net_buffer. The word “buffer”, if used by itself, will be qualified by its context - it may not always refer to a network buffer.
```

Comment:
s/M0_net_buffer/m0_net_buffer/g

Please check and correct all of them.

Reply:
It's better to have "Network Buffer" using the font **BOLD**, to match the original document.

So, to keep the document more readable, we need more beautiful page format.

Reply:
Incorporated.